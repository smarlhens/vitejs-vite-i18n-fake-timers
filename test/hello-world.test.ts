import type { RenderResult } from '@testing-library/vue';
import { render } from '@testing-library/vue';
import { afterEach, beforeEach, describe, expect, it } from 'vitest';

import HelloWorld from '../src/components/hello-world.vue';

describe('HelloWorld', () => {
    let wrapper: RenderResult;

    afterEach(() => {
        wrapper.unmount();
    });

    beforeEach(() => {
        wrapper = render(HelloWorld);
    });

    it('should render', () => {
        const { html, getByText } = wrapper;
        getByText('HelloWorld');
        expect(html()).toMatchSnapshot();
    });
});
