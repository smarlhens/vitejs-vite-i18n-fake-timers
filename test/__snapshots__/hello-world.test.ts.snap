// Vitest Snapshot v1

exports[`HelloWorld > should render 1`] = `"<h1>HelloWorld</h1>"`;
